ebos/ecloutputblackoilmodule.hh

```diff
@@ -1149,6 +1160,14 @@ public:
         return 0;
     }
 
+    Scalar getFoamConcentration(unsigned elemIdx) const
+    {
+        if (cFoam_.size() > 0)
```

Comment:
may as well check that size() > elemIdx;

Reply:
I had to think about this one. With the current code, a nonzero size <= elemIdx could trigger a segfault alerting us to an error, while with your suggestion would be more robust, hiding the error by returning zero. In the end I'm going with your suggestion, robustness is better.